src/_pytest/python.py

```diff
@@ -1136,32 +1132,23 @@ class CallSpec2:
     def setmulti(
         self,
         *,
-        valtypes: Mapping[str, "Literal['params', 'funcargs']"],
```

Comment:
at first glance this looks like a change that might break some useages, do i recall correct that we dont officially export this apu

Reply:
So it won't be a problem? Given that it hasn't been exposed to the user.

Reply:
@bluetech , by pytest private API you mean the code in pytest-dev repositories? As in this repo, just in one place `setmulti` is used.

Reply:
The public pytest API is broadly everything that is listed in the API Reference: https://docs.pytest.org/en/latest/reference/reference.html. These APIs cannot be changed without deprecation and a major release. 

The "private" API is everything else; ideally only pytest itself would use it.

Over time we've worked on improving the encapsulation, exposing & documenting what is public and more aggressively hiding what is private. But this is work in progress and not always clear. In ambiguous cases we look at actual usage. In the case of `setmulti` it seems OK to change.